### PR TITLE
Tasks/blog 68 product validation

### PR DESCRIPTION
**Pull request by hoavu**

### CODE REVIEW

File: source/_posts/2016-10-04-cach-tiep-can-nhanh-de-tham-dinh-san-pham.markdown [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    - "trans"
+tags:
+    - "product"
+    - "validation"
+cover: /images/2016/10/04/thamdinhnhanh.jpg
+description: Cách tiếp cận thẩm định sản phẩm giúp bạn xây dựng được sản phẩm đúng như mong muốn
+keywords: Product Strategy, Product Validation
+---
+
+{% img center /images/2016/10/04/thamdinhnhanh.jpg Thẩm định sản phẩm nhanh %}
```
Comment (hoatle) [inline source/_posts/2016-10-04-cach-tiep-can-nhanh-de-tham-dinh-san-pham.markdown]: tham-dinh-san-pham-nhanh.jpg

File: source/_posts/2016-10-04-cach-tiep-can-nhanh-de-tham-dinh-san-pham.markdown [hunk 2]
```diff
@@ -0,0 +1,124 @@
+---
```
Comment (hoatle) [inline source/_posts/2016-10-04-cach-tiep-can-nhanh-de-tham-dinh-san-pham.markdown]: the file name: 2016-10-04-cach-tiep-can-nhanh-de-tham-dinh-san-pham-phan-1.markdown

File: source/_posts/2016-10-04-cach-tiep-can-nhanh-de-tham-dinh-san-pham.markdown [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+Trong phần hướng dẫn ngắn này, bạn sẽ học được cách thẩm định sự phù hợp của sản phẩm với thị trường, khảo sát người dùng và sản phẩm dùng thử. Mục đích để đảm bảo ý tưởng sản phẩm đó khả thi, điều này giúp tiết kiệm thời gian và tiền bạc của bạn.
+
+<!-- more -->
+
+Cuối quá trình, bạn sẽ có thể tập trung vào việc xây dựng sản phẩm và tự tin rằng bạn có cơ hội thành công cao hơn!
+
+Bạn đã sẵn sang chưa? Chúng ta bắt đầu nhé.
+
+{% img center /images/2016/10/04/thamdinhnhanh.jpg Thẩm định sản phẩm nhanh %}
```
Comment (hoatle) [inline source/_posts/2016-10-04-cach-tiep-can-nhanh-de-tham-dinh-san-pham.markdown]: tham-dinh-san-pham-nhanh.jpg

File: source/_posts/2016-10-04-cach-tiep-can-nhanh-de-tham-dinh-san-pham.markdown [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+comments: true
+categories:
+    - "vi"
+    - "trans"
+tags:
+    - "product"
+    - "validation"
+cover: /images/2016/10/04/thamdinhnhanh.jpg
+description: Cách tiếp cận thẩm định sản phẩm giúp bạn xây dựng được sản phẩm đúng như mong muốn
+keywords: Product Strategy, Product Validation
```
Comment (hoatle) [inline source/_posts/2016-10-04-cach-tiep-can-nhanh-de-tham-dinh-san-pham.markdown]: add Vietnamese keywords, too

File: source/_posts/2016-10-04-cach-tiep-can-nhanh-de-tham-dinh-san-pham.markdown [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+**3. Thẩm định sản phẩm**
+
+Vấn đề có thể tồn tại nhưng sản phẩm của bạn có thực sự xử lí được vấn đề đó?
+
+**4. Thẩm định sự sẵn lòng chi trả**
+
+Có thể thị trường có nhu cầu và sản phẩm của bạn rất tuyệt vời. Nhưng liệu mọi người có sẵn sằng trả tiền mua sản phẩm đó không?
+
+
+{% img center /images/2016/10/04/validationprocess.jpg Quy trình thẩm định sản phẩm  %}
```
Comment (hoatle) [inline source/_posts/2016-10-04-cach-tiep-can-nhanh-de-tham-dinh-san-pham.markdown]: quy-trinh-tham-dinh-san-pham.jpg

File: source/_posts/2016-10-04-cach-tiep-can-nhanh-de-tham-dinh-san-pham.markdown [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+ 
+ĐIỀU GÌ XẢY RA NẾU Ý TƯỞNG CỦA BẠN ĐÁP ỨNG TẤT CẢ BỐN TIÊU CHÍ TRÊN?
```
Comment (hoatle) [inline source/_posts/2016-10-04-cach-tiep-can-nhanh-de-tham-dinh-san-pham.markdown]: the heading is so inconsistent, pls make theme consistent.

File: source/_posts/2016-10-04-cach-tiep-can-nhanh-de-tham-dinh-san-pham.markdown [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+Cho dù bạn có tự tin về ý tưởng sản phẩm như nào chăng nữa thì điều đầu tiên bạn cần là xác định liệu vấn đề đó có thực sự cần giải quyết. Để làm được điều này, lựa chọn tốt nhất của bạn là nói chuyện trực tiếp với người sử dụng tiềm năng. Trọng tâm ở đây là về việc đánh giá chất lượng các ý tưởng sản phẩm.
+
+Đầu tiên, chúng ta bắt đầu với một số lượng nhỏ người dùng đại diện phù hợp và xác định xem vấn đề đó họ có gặp phải không. Sau đó chúng ta [tối ưu hóa và xác định](https://leanstack.com/pivot-before-productmarket-fit-optimize-after/) vấn đề này với nhiều người dùng hơn và ở quy mô lớn hơn.
+
+Những điều bạn thu được từ các cuộc phỏng vấn này sẽ giúp bạn tự tin để thúc đẩy ý tưởng của mình tiến xa hơn. Hoặc nó cũng sẽ cho bạn thấy rằng bạn cần phải đào sâu ý tưởng khi bạn tìm hiểu thêm những vấn đề cơ bản thực sự.
+
+Ba kỹ thuật được miêu tả dưới đây có thể dùng riêng biệt, nhưng chúng tạo thành một bộ các phương pháp có thể cung cấp một hình ảnh tốt hơn về vấn đề mà bạn đang cố gắng thẩm định.
+
+
+{% img center /images/2016/10/04/validateproblem.jpg Thẩm định vấn đề %}
```
Comment (hoatle) [inline source/_posts/2016-10-04-cach-tiep-can-nhanh-de-tham-dinh-san-pham.markdown]: tham-dinh-van-de.jpg

**Comment by hoatle** [comment]
@hoavu review done, pls check my comments.

File: source/_posts/2016-10-04-cach-tiep-can-nhanh-de-tham-dinh-san-pham.markdown (hunk 6 above)
Comment (hoavu) [inline source/_posts/2016-10-04-cach-tiep-can-nhanh-de-tham-dinh-san-pham.markdown]: Keep the heading as the original blog

**Pull request merged**